### DISCUSSION TITLE
SAVING BUTTON

### DISCUSSION THREAD

**Discussion by Lunaire333** [AmyE123/TactiKit#6] [Map Editor Feedback]
For the save option. It was great to mention that the disappearance of the asterisk means that the project has been saved. However, what would also be better is to have the option of an auto-save button/function. 

It is common for artists/designers to forget to save their projects, especially when they are in their momentum. So, having the "auto-save" function would save them much time and make the experience less stressful. 

![image](https://github.com/AmyE123/TactiKit/assets/158181584/25bf1a5e-d5c1-44e1-bf77-79ba8403920a)

**Comment by AmyE123** [comment]
Would this autosave function be enabled straight away, or would it have to be enabled in some settings screen?

**Reply by Lunaire333** [reply]
I think enabling the autosave function in some settings screens would be much better:
- The user can have a sense of control in their project
- The user may not prefer an autosave function because he wants to be sure he saved his projects manually.

**Reply by bLInVoid** [reply]
You could maybe also add an autosave interval for the user? Just in case someone usually prefers to manually save, but wants a sort of failsafe if they forget. Or if not that then a popup could appear when closing the toolkit asking if they want to save upon exit, just to eliminate any chance of anyone losing progress
